index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First MicroVerse Push</title>
```

Comment:
- [ ] Kindly add a link here for the CSS file after this code. :arrow_up: 

- Just do it like this: `<link rel="stylesheet" href="first.css">`